CMSgov/mint-app
# [EASI-4481] Organize SQL Query Files

# EASI-4481

## Description

We are updating our organizational structure to better maintain SQL throughout our codebase. As we move these files, we must create corresponding structures to contain script references which can then be referenced throughout the codebase.


## How to test this change

- Open your IDE and confirm all SQL scripts have been moved to `pkg/sqlqueries/SQL/{query_package}`
- Build and run server containers
- Smoke test server to assert no functionality has been changed

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
